pkg/handler/payload.go

```diff
@@ -0,0 +1,179 @@
+package handler
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/instill-ai/pipeline-backend/config"
+	"github.com/instill-ai/pipeline-backend/internal/constant"
+	"github.com/instill-ai/pipeline-backend/pkg/service"
+)
+
+func parseImageFormDataInputsToBytes(req *http.Request) (visionInput *service.VisionInput, err error) {
+
+	inputs := req.MultipartForm.File["file"]
+	content := make([]byte, 0, len(inputs))
+	fileNames := make([]string, 0, len(inputs))
+	fileLengths := make([]uint64, 0, len(inputs))
+	for _, input := range inputs {
+		file, err := input.Open()
+		defer func() {
+			err = file.Close()
+		}()
+
+		if err != nil {
+			return nil, fmt.Errorf("Unable to open file for image")
+		}
+
+		buff := new(bytes.Buffer)
+		numBytes, err := buff.ReadFrom(file)
+		if err != nil {
+			return nil, fmt.Errorf("Unable to read content body from image")
+		}
+		if numBytes > int64(config.Config.Server.MaxDataSize*constant.MB) {
+			return nil, fmt.Errorf(
+				"Image size must be smaller than %vMB. Got %vMB",
+				config.Config.Server.MaxDataSize,
+				float32(numBytes)/float32(constant.MB),
+			)
+		}
+
+		content = append(content, buff.Bytes()...)
+		fileNames = append(fileNames, input.Filename)
+		fileLengths = append(fileLengths, uint64(buff.Len()))
+	}
+	return &service.VisionInput{
+		Content:     content,
+		FileNames:   fileNames,
+		FileLengths: fileLengths,
+	}, nil
+}
+
+func parseImageFormDataTextToImageInputs(req *http.Request) (textToImageInput *service.TextToImageInput, err error) {
+	prompts := req.MultipartForm.Value["prompt"]
+	if len(prompts) == 0 {
+		return nil, fmt.Errorf("missing prompt input")
+	}
+	if len(prompts) > 1 {
+		return nil, fmt.Errorf("invalid prompt input, only support a single prompt")
+	}
+	stepStr := req.MultipartForm.Value["steps"]
+	cfgScaleStr := req.MultipartForm.Value["cfg_scale"]
+	seedStr := req.MultipartForm.Value["seed"]
+	samplesStr := req.MultipartForm.Value["samples"]
+
+	if len(stepStr) > 1 {
+		return nil, fmt.Errorf("invalid steps input, only support a single steps")
+	}
+	if len(cfgScaleStr) > 1 {
+		return nil, fmt.Errorf("invalid cfg_scale input, only support a single cfg_scale")
+	}
+	if len(seedStr) > 1 {
+		return nil, fmt.Errorf("invalid seed input, only support a single seed")
+	}
+	if len(samplesStr) > 1 {
+		return nil, fmt.Errorf("invalid samples input, only support a single samples")
+	}
+
+	step := 10
```

Comment:
Please move these default constants to `internal/constant/constant.go` with comments.